phc-rails/config/routes.rb

```diff
@@ -9,6 +9,7 @@
       get "accounts/:sf_id", to: "accounts#show"
       post 'event_registrations/create'
       get 'event_registrations/search'
+      get 'event_registrations/get_applied'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.